resolwe/flow/tasks.py

```diff
@@ -7,11 +7,16 @@
 """
 from __future__ import absolute_import
 
+import subprocess
+
 from celery import shared_task  # pylint: disable=import-error
 
 
 @shared_task
-def celery_run(data_id, script, verbosity):
+def celery_run(data_id, dest_dir, argv, verbosity):
     """Run process executor."""
```

Comment:
Please, document `celery_run` function's arguments in its docstring.

Reply:
I left this docstring as it was pre-refactor.